### DISCUSSION TITLE
My student email shows the wrong school

### DISCUSSION THREAD

**Discussion by y-u-s-u-f** [community/community#99104] [GitHub Education]
### Hi everyone,

Hi!

I was trying to apply for the pack, but my student email shows the wrong school. I applied using my gmail email and I got 

> Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Jan 19, 2024 for @YufusKhan.  Attendees of this school must verify and use their academic email address. If your school does not provide academic email then you must enable use of your device camera to capture your image. Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy. Your image contains keywords indicating that it may be an acceptance letter. Only proof of current academic enrollment is acceptable.

I just submitted it again but with the student email even if it shows the wrong email, is this OK? Should I contact support?

**Comment by muslimbek77** [comment]
It seems like there might be a verification issue with the academic affiliation process. In this case, it would be advisable to contact support for assistance. They can provide you with guidance on how to proceed and help resolve any issues with the verification process.

When contacting support, make sure to explain the situation, including the incorrect school information associated with your student email. Provide them with any relevant details and documentation that can help verify your academic affiliation accurately.

Additionally, double-check your submission to ensure that you've followed the correct steps and included the necessary information. If there are any specific instructions or requirements outlined by the platform you are applying to, make sure you've adhered to them.

Contacting support is the best way to address these types of issues and ensure a smooth resolution to the verification process.

**Reply by y-u-s-u-f** [reply]
Thanks, will do!
